docs/reST/ref/display.rst

```diff
@@ -472,7 +472,10 @@ required).
    True if successful.
 
    When the display is iconified ``pygame.display.get_active()`` will return
-   False. The event queue should receive a ``ACTIVEEVENT`` event when the
+   False. When using SDL 1 (usually in pygame versions less than 2.0.0) the
```

Comment:
I wonder if this backwards compatibility should be captured in an issue?

I think it's easier for users to read the wording pygame 2, rather than SDL1 or SDL2. Since understanding SDL versions is maybe above the heads of most people.